raiden-dapp/tests/unit/utils/stub.ts

```diff
@@ -1,6 +1,7 @@
 export function stub<T>(): T {
   const typeAssertion = <T>{};
   for (const prop in typeAssertion) {
+    // @ts-ignore
     if (typeAssertion.hasOwnProperty(prop)) {
```

Comment:
This is the same as:
```suggestion
     if (prop in typeAssertion) {
```
But I actually don't understand the point, can you clarify? First, all `const prop in typeAssertion` already are `in typeAssertion`. But despite the type T you declared, at runtime `typeAssertion` is an empty object, so the for loop is noop. In practice, at runtime this function just return an empty object, and on TypeScript it may be problematic because you _tell_ TypeScript it'll be an object of type T but empty object may not comply with the type, and TS will never know it.

Reply:
Theoretically, this was supposed to create a stub object with all the properties set to undefined. In practice what you mentioned happened. I also realized that this was used only in one place so no need for the extra function anymore, so I am removing it since it does not make any sense to keep a function for only one place.